CONTRIBUTING.md

```diff
@@ -1,6 +1,28 @@
-# Writing the Bevy book
+# Contributing
 
-So, you want to help write some learning material for the [Bevy website](https://bevyengine.org/)?
+So, you want to help with the [Bevy website](https://bevyengine.org/)?
+
+## Building the website
+
+The Bevy website is built using the Zola static site engine. In our experience, it is fast, flexible, and straightforward to use.
+
+To check out any local changes you've made:
+
+1. [Download Zola](https://www.getzola.org/).
+2. Clone the Bevy Website git repo and enter that directory:
+    1. `git clone https://github.com/bevyengine/bevy-website.git`
+    2. `cd bevy-website`
+3. Start the Zola server with `zola serve`.
+
+A local server should start and you should be able to access a local version of the website from there.
+
+## Testing your changes
```

Comment:
IMO this heading should express that this paragraph is only about code snippets in the book.

Reply:
```suggestion
## Testing changes to book code snippets
```